### DISCUSSION TITLE
About the About Page

### DISCUSSION THREAD

**Discussion by cindyorangis** [Seneca-CDOT/telescope#2985] [Ideas]
I have a confession to make; years ago, I copied and pasted the [Overview doc](https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/overview.md) and placed it into the "About Us" page. We need a real About/About Us page that defines us/Telescope. This should be done before we start our translation work on #2962.

Questions we might want to address:
- What do we want to say about us or Telescope?
- What do we want people to know about Telescope?
- What are we trying to achieve with Telescope?
- more

**Comment by JerryHue** [comment]
Well, for starters, the work that Diana is doing (#3149) is going to be added on the top of the about page, so maybe we can start to write the About Us page basing off of that?

**Comment by tpmai22** [comment]
We also need to fix broken links in our about page
